modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ModalAddObjectField.tsx

```diff
@@ -236,25 +239,28 @@ export function ModalAddObjectField({
 									values={values}
 								/>
 							)}
-						</ClayModal.Body>
+						</ClayForm>
+					</ClayModal.Body>
 
-						<ClayModal.Footer
-							last={
-								<ClayButton.Group spaced>
-									<ClayButton
-										displayType="secondary"
-										onClick={() => onClose()}
-									>
-										{Liferay.Language.get('cancel')}
-									</ClayButton>
+					<ClayModal.Footer
+						last={
+							<ClayButton.Group spaced>
+								<ClayButton
+									displayType="secondary"
+									onClick={() => onClose()}
+								>
+									{Liferay.Language.get('cancel')}
+								</ClayButton>
 
-									<ClayButton type="submit">
-										{Liferay.Language.get('save')}
-									</ClayButton>
-								</ClayButton.Group>
-							}
-						/>
-					</ClayForm>
+								<ClayButton
+									form="modalAddObjectField"
```

Comment:
@Artartur one other small thing... It's generally good practice to create a variable for an `id` like this since it's used in more than one place. You can then use the variable in both places. This makes it easier to understand that it needs to be the same and makes maintaining the code easier.

Reply:
Yeah, it makes sense, thx @ethib137